automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/notebooks/WorkspaceNotebooksPage.scala

```diff
@@ -0,0 +1,207 @@
+package org.broadinstitute.dsde.firecloud.page.workspaces.notebooks
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.FireCloudView
+import org.broadinstitute.dsde.firecloud.component.Component._
+import org.broadinstitute.dsde.firecloud.component._
+import org.broadinstitute.dsde.firecloud.page.PageUtil
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspacePage
+import org.broadinstitute.dsde.workbench.config.Config
+import org.broadinstitute.dsde.workbench.model.google.GcsPath
+import org.broadinstitute.dsde.workbench.service.test.WebBrowserUtil
+import org.openqa.selenium.WebDriver
+import org.scalatest.selenium.{Page, WebBrowser}
+
+class WorkspaceNotebooksPage(namespace: String, name: String)(implicit webDriver: WebDriver)
+  extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceNotebooksPage] with LazyLogging {
+
+  override def awaitReady(): Unit = clustersTable.awaitReady()
+
+  override val url: String = s"${Config.FireCloud.baseUrl}#workspaces/$namespace/$name/notebooks"
+
+  private val clustersTable = Table("spark-clusters-table")
+  private val clustersTableTestId = testId("spark-clusters-table")
+  private val clusterErrorMessage = testId("notebooks-error")
+  private val sparkClustersHeader = testId("spark-clusters-title")
+  private val openCreateClusterModalButton: Button = Button("create-modal-button")
+  private val noClustersMessage = "There are no clusters to display."
+
+  //  withNewCluster - open modal, create cluster, wait for it to appear/running state, open delete modal, delete cluster, wait for delete
+  def withNewCluster(clusterName: String,
+                     masterMachineType: String = "n1-standard-4",
+                     masterDiskSize: Int = 500,
+                     workers: Int = 0,
```

Comment:
Specified the defaults here - maybe I should put them somewhere else?

Reply:
seems ok here